spec/rails_helper.rb

```diff
@@ -41,6 +41,11 @@
     page.driver.browser.manage.window.resize_to(1280,3072)
   end
 
+  config.before(:each) do
+    # Stop various test setups from indirectly attempting to push handover dates to the Community API
```

Comment:
This seems too agressive - tagging the relevant ones (see POM-81) would be much less invasive